packages/fxa-auth-server/lib/senders/emails/css/style.css

```diff
@@ -0,0 +1,32 @@
+.header-text div {
+  font-family: sans-serif !important;
+  font-size: 20px !important;
+  line-height: 28px !important;
+  font-weight: normal !important;
+  text-align: center !important;
+}
+
+td {
+  padding: 4px !important;
+}
+
+span {
+  color: #0a84ff;
+}
```

Comment:
We'll need to be careful with writing too-broad selectors in global files. For example, here, is every `span` across every template going to have this color? Or above, is every `td` going to have a 4px padding? We should consider using classes more generously so we're not painting ourselves into a corner with far-reaching selectors.

Reply:
While converting `mjml` into `html`, MJML adds its own styles inlined with the html elements so if we're adding our custom styles on top of that and they are overlapping, they will get ignored unless we add `!important` and then our custom styles get prioritized. While, I do agree that `span` might not be used in all templates, `td` is something that needs to be taken care of everywhere. I observed that majority of mjml elements get narrowed down to `td` and each one of it has its own padding of around 10-25 px which looks weird in our emails. If adding `!important` to each and every style is not looking neat then we can also use mjml attributes and do something like [this](https://mjml.io/try-it-live/5JTenq09jXb).